rails_event_store_active_record/lib/rails_event_store_active_record/generators/templates/no_global_stream_entries_template.rb

```diff
@@ -0,0 +1,48 @@
+class NoGlobalStreamEntries < ActiveRecord::Migration<%= migration_version %>
+  def change
+    case ActiveRecord::Base.connection.adapter_name
+    when "SQLite"
+      rename_table :event_store_events, :old_event_store_events
+      create_table(:event_store_events, force: false) do |t|
+        t.references  :event,       null: false, type: :string, limit: 36
+        t.string      :event_type,  null: false
+        t.binary      :metadata
+        t.binary      :data,        null: false
+        t.datetime    :created_at,  null: false
+      end
+      add_index :event_store_events, :event_id, unique: true
+      add_index :event_store_events, :created_at
+      add_index :event_store_events, :event_type
+
+      execute <<-SQL
+        INSERT INTO event_store_events(event_id, event_type, metadata, data, created_at)
+        SELECT id, event_type, metadata, data, created_at FROM old_event_store_events;
+      SQL
+      drop_table :old_event_store_events
+    when "PostgreSQL"
+      rename_column :event_store_events, :id, :event_id
+      change_column_default :event_store_events, :event_id, nil
```

Comment:
So `event_id` must be always provided by the client.

Reply:
Indeed. We never relied on it in our test cases and this default was not symmetrical with mysql/sqlite. 

Also: https://github.com/RailsEventStore/rails_event_store/blob/master/ruby_event_store/lib/ruby_event_store/event.rb#L15